uk.co.compucorp.civicrm.hrleaveandabsences/js/test/shared/modules/constants.spec.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'leave-absences/shared/modules/constants'
+], function () {
+  'use strict';
+
+  describe('constants', function () {
+    var absenceTypeColours;
+
+    beforeEach(module('leave-absences.constants'));
+    beforeEach(inject(['absence-type-colours', function (_absenceTypeColours_) {
+      absenceTypeColours = _absenceTypeColours_;
+    }]));
+
+    describe('absence types colours', function () {
```

Comment:
I don't think there was a need to test the constants, but ok 🤷‍♂️

Reply:
What if someone adds a constant that does not comply with requirements?

Reply:
typing `red` for example? If that's going to break the code, then it's fine. Though I doubt this would happen. But since it's already here, lets leave it.